x-pack/plugins/apm/public/hooks/use_fetcher.test.tsx

```diff
@@ -32,6 +32,34 @@ function wrapper({ children }: WrapperProps) {
 }
 
 describe('useFetcher', () => {
+  describe('when data fetch is dropped', () => {
+    let hook: RenderHookResult<
+      WrapperProps,
+      FetcherResult<unknown> & {
+        refetch: () => void;
+      }
+    >;
+
+    beforeEach(() => {
+      hook = renderHook(
+        () =>
+          useFetcher(() => {
+            return;
+          }, []),
+        { wrapper }
+      );
+    });
+
+    it('should update status to dropped', async () => {
+      expect(hook.result.current).toEqual({
+        data: undefined,
+        error: undefined,
+        refetch: expect.any(Function),
+        status: 'dropped',
```

Comment:
I don't think this should have a status of dropped. Looks like `NOT_INITIATED` to me